src/openrct2/ride/transport/meta/Monorail.h

```diff
@@ -16,30 +16,32 @@
-    SET_FIELD(DefaultMode, RIDE_MODE_CONTINUOUS_CIRCUIT),
-    SET_FIELD(Naming, { STR_RIDE_NAME_MONORAIL, STR_RIDE_DESCRIPTION_MONORAIL }),
-    SET_FIELD(NameConvention, { RIDE_COMPONENT_TYPE_TRAIN, RIDE_COMPONENT_TYPE_TRACK, RIDE_COMPONENT_TYPE_STATION }),
-    SET_FIELD(AvailableBreakdowns, (1 << BREAKDOWN_SAFETY_CUT_OUT) | (1 << BREAKDOWN_DOORS_STUCK_CLOSED) | (1 << BREAKDOWN_DOORS_STUCK_OPEN) | (1 << BREAKDOWN_VEHICLE_MALFUNCTION)),
-    SET_FIELD(Heights, { 8, 32, 8, 9, }),
-    SET_FIELD(MaxMass, 78),
-    SET_FIELD(LiftData, { SoundId::Null, 5, 5 }),
-    SET_FIELD(UpkeepCosts, { 65, 20, 0, 10, 3, 10 }),
-    SET_FIELD(BuildCosts, { 42, 5, 50, }),
-    SET_FIELD(DefaultPrices, { 10, 0 }),
-    SET_FIELD(DefaultMusic, MUSIC_STYLE_SUMMER),
-    SET_FIELD(PhotoItem, SHOP_ITEM_PHOTO),
+  SET_FIELD(AlternateType, RIDE_TYPE_NULL),
+  SET_FIELD(Category, RIDE_CATEGORY_TRANSPORT),
+  SET_FIELD(EnabledTrackPieces, (1ULL << TRACK_STRAIGHT) | (1ULL << TRACK_STATION_END) | (1ULL << TRACK_SLOPE) | (1ULL << TRACK_S_BEND) | (1ULL << TRACK_CURVE_SMALL) | (1ULL << TRACK_CURVE)),
+  SET_FIELD(ExtraTrackPieces, 0),
+  SET_FIELD(StartTrackPiece, TRACK_ELEM_END_STATION),
+  SET_FIELD(TrackPaintFunction, get_track_paint_function_monorail),
+  SET_FIELD(Flags, RIDE_TYPE_FLAG_HAS_TRACK_COLOUR_MAIN | RIDE_TYPE_FLAG_HAS_TRACK_COLOUR_ADDITIONAL | RIDE_TYPE_FLAG_CAN_SYNCHRONISE_ADJACENT_STATIONS |
+           RIDE_TYPE_FLAG_HAS_TRACK_COLOUR_SUPPORTS | RIDE_TYPE_FLAG_HAS_LEAVE_WHEN_ANOTHER_VEHICLE_ARRIVES_AT_STATION |
+           RIDE_TYPE_FLAG_HAS_DATA_LOGGING | RIDE_TYPE_FLAG_HAS_LOAD_OPTIONS | RIDE_TYPE_FLAG_HAS_VEHICLE_COLOURS |
+           RIDE_TYPE_FLAG_HAS_TRACK | RIDE_TYPE_FLAG_HAS_LARGE_CURVES | RIDE_TYPE_FLAG_SUPPORTS_MULTIPLE_TRACK_COLOUR |
+           RIDE_TYPE_FLAG_ALLOW_MUSIC | RIDE_TYPE_FLAG_HAS_ENTRANCE_EXIT | RIDE_TYPE_FLAG_ALLOW_MORE_VEHICLES_THAN_STATION_FITS |
+           RIDE_TYPE_FLAG_ALLOW_MULTIPLE_CIRCUITS | RIDE_TYPE_FLAG_TRANSPORT_RIDE | RIDE_TYPE_FLAG_SHOW_IN_TRACK_DESIGNER),
+  SET_FIELD(RideModes, (1ULL << RIDE_MODE_CONTINUOUS_CIRCUIT) | (1ULL << RIDE_MODE_SHUTTLE)),
+  SET_FIELD(DefaultMode, RIDE_MODE_CONTINUOUS_CIRCUIT),
+  SET_FIELD(OperatingSettings, { 0, 0, 0, 0, 0, 0 }),
+  SET_FIELD(Naming, { STR_RIDE_NAME_MONORAIL, STR_RIDE_DESCRIPTION_MONORAIL }),
+  SET_FIELD(NameConvention, { RIDE_COMPONENT_TYPE_TRAIN, RIDE_COMPONENT_TYPE_TRACK, RIDE_COMPONENT_TYPE_STATION }),
+  SET_FIELD(AvailableBreakdowns, (1 << BREAKDOWN_SAFETY_CUT_OUT) | (1 << BREAKDOWN_DOORS_STUCK_CLOSED) | (1 << BREAKDOWN_DOORS_STUCK_OPEN) | (1 << BREAKDOWN_VEHICLE_MALFUNCTION)),
+  SET_FIELD(Heights, { 8, 32, 8, 9, }),
+  SET_FIELD(MaxMass, 78),
+  SET_FIELD(LiftData, { SoundId::Null, 5, 5 }),
+  SET_FIELD(UpkeepCosts, { 65, 20, 0, 10, 3, 10 }),
+  SET_FIELD(BuildCosts, { 42, 5, 50, }),
+  SET_FIELD(DefaultPrices, { 10, 0 }),
+  SET_FIELD(DefaultMusic, MUSIC_STYLE_SUMMER),
+  SET_FIELD(PhotoItem, SHOP_ITEM_PHOTO),
+  SET_FIELD(BonusValue, 60)
```

Comment:
Why did you change the indentation?

Reply:
That find and replace just screwed me up.